docs/native-modules.md

```diff
@@ -312,21 +320,24 @@ This can be also tied in with C++/WinRT event handlers or `IAsyncOperation<T>` l
 
 ```cpp
     REACT_METHOD(GetString, L"getString");
-    void GetString(bool error, React::ReactPromise<React::JSValue>&& result) noexcept
+    void GetString(React::ReactPromise<std::string>&& result) noexcept
     {
       if (error) {
         result.Reject("Failure");
       } else {
         something.Completed([result] (const auto& operation, const auto& status) {

```

Comment:
>something.Completed [](start = 8, length = 19)

Let's use the co-routine syntax instead.

Reply:
filed 283 to add an example using coroutines